doc/oper.xml

```diff
@@ -2131,3 +2133,47 @@ gap> data;
 </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="IsDigraphPath">
+<ManSection>
+  <Oper Name="IsDigraphPath" Arg="D, v, a"/>
```

Comment:
I’m happy for this version of arguments to exist but I think that `IsDigraphPath` should also (and in my mind this would be the main one) have a version that accepts the output of `DigraphPath` as input. ie if there was a path from 1 to 2 in D then `IsDigraphPath(D, DigraphPath(D, 1, 2));` would return true.

Reply:
Done, although I left the main one as being the one that accepts 3 args, apart from anything, to avoid creating a list where one isn't required. Happy to revisit if necessary!